guides/common/modules/proc_installing-and-configuring-puppet-agent-manually.adoc

```diff
@@ -64,7 +61,7 @@ server = _{foreman-example-com}_
 # chkconfig puppet on
 ----
 ifdef::satellite[]
-* On {EL} 7 and 8:
+* On {EL} 7, 8, and 9:
```

Comment:
Would it make sense to write 7 and beyond? I think it's fairly safe to assume EL 7+ will always use systemd.